test/test_backend.py

```diff
@@ -23,6 +23,7 @@
 
 TEST_DIR = os.path.dirname(__file__)
 BACKEND_URL_BASE = 'http://127.0.0.1:8000/'
+COUNT_TEST_EXCEPTIONS = ['concept_set_json', 'rxnorm_med_cset', 'small_snomed']
```

Comment:
Oh, FYI, I capitalized `COUNT_TEST_EXCEPTIONS`. Just wanted to explain why I did that. It's sometimes a convention that whenever a variable is in "global scope" (i.e. outside of any function or class), it should be treated as a constant and should not be changed at run-time. This helps minimize bugs, and capitalizing it helps people know that it is a constant. This variable is in the global scope because it is shared by more than 1 function.

Reply:
Ok, thank you! We were wondering why some cells in the dataframe had value 0.

Reply:
That makes sense !